apple/swift-docc-render
# Add banner that suggest to switch to preferred locale by user

Bug/issue #104924748, if applicable: 

## Summary

Add banner that suggest to switch to preferred locale by user

## Dependencies

NA

## Testing

Steps:
1. Add the following to `theme-settings.json` file at public folder
```json
{
  "features": {
    "docs": {
      "i18n": {
        "enable": true
      }
    }
  }
}
```
3. Run `npm run serve`
2. Go to the bottom of the page and change the language in the locale selector Chinese for example
3. Delete localStorage
4. If you have your browser in English, you should assert that a banner is displayed

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
